.gitignore

```diff
@@ -7,3 +7,6 @@
 /*.scss
 /*.sass
 /*.less
+
+# IDE
+.idea/
```

Comment:
These should also go in your global gitignore

Reply:
mission complete))